src/components/dashboard/SalesMetricCard.tsx

```
@@ -0,0 +1,428 @@
+'use client'
+
+import { ArrowDown, ArrowUp, DollarSign, Percent, TrendingUp } from 'lucide-react'
+import React from 'react'
+
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card'
+import { Skeleton } from '@/components/ui/skeleton'
+import { formatBRL, formatPercentage } from '@/lib/analytics-utils'
```

ESLint

[error] 'formatPercentage' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)